roles/workshop-spawner/defaults/main.yml

```diff
@@ -2,8 +2,8 @@
 workshop_spawner_project_name: devsecops
 workshop_spawner_project_display: DevSecOps Common Resources
 
-workshop_spawner_image: quay.io/akrohg/workshop-spawner:latest
-workshop_dashboard_image: quay.io/akrohg/devsecops-workshop-dashboard:latest
+workshop_spawner_image: quay.io/redhatgov/workshop-spawner:latest
+workshop_dashboard_image: quay.io/redhatgov/openshift-devsecops-labguides:{{ pipeline_style|default("tekton") }}
```

Comment:
Considering that these are a separate artifact, going forward they should probably be versioned in some meaningful way (beyond 'tekton' vs 'jenkins').

Reply:
Agree, this is probably for the best. Perhaps we should open an issue about version pinning all of the components once we have finished the rapid-iteration development and work out how to appropriately separate development and production copies of the workshop?

Reply:
(version-pinning all components will require that we work out how best to version the lab guides as well)